src/rc/seedrng.c

```diff
@@ -404,17 +462,17 @@ int main(int argc _unused, char *argv[] _unused)
 	blake2s_update(&hash, &realtime, sizeof(realtime));
 	blake2s_update(&hash, &boottime, sizeof(boottime));
 
-	if (mkdir(SEED_DIR, 0700) < 0 && errno != EEXIST)
-		eerrorx("Unable to create \"%s\" directory: %s", SEED_DIR, strerror(errno));
+	if (mkdir(seed_dir, 0700) < 0 && errno != EEXIST)
+		eerrorx("Unable to create \"%s\" directory: %s", seed_dir, strerror(errno));
 
-	lock = open(LOCK_FILE, O_WRONLY | O_CREAT, 0000);
+	lock = open(lock_file, O_WRONLY | O_CREAT, 0000);
 	if (lock < 0 || flock(lock, LOCK_EX) < 0)
 		eerrorx("Unable to open lock file: %s", strerror(errno));
 
-	ret = seed_from_file_if_exists(NON_CREDITABLE_SEED, false, &hash);
+	ret = seed_from_file_if_exists(non_creditable_seed, false, &hash);
 	if (ret < 0)
 		program_ret |= 1 << 1;
-	ret = seed_from_file_if_exists(CREDITABLE_SEED, !rc_yesno(getenv("SEEDRNG_SKIP_CREDIT")), &hash);
+	ret = seed_from_file_if_exists(creditable_seed, !rc_yesno(getenv("SEEDRNG_SKIP_CREDIT")), &hash);
```

Comment:
If you choose to keep the option parsing and get rid of the getenv, maybe also turn SEEDRNG_SKIP_CREDIT into an option too?

Or get rid of the option parsing, and do it all with environment variables.

Reply:
@williamh It looks like you've added another option, but are keeping the environment variable thing. Can you tell me why you want to have _two ways_ of configuring the same utility? Just choose one. It doesn't matter which one. But having two ways to do it will lead to hard to debug problems for somebody eventually.